__mocks__/browser-request.js

```diff
@@ -1,4 +1,5 @@
 const en = require("../src/i18n/strings/en_EN");
+const de = require("../src/i18n/strings/de_DE");
```

Comment:
this & below are debug?

Reply:
Nope, this is fixing the German test which always failed but given it was async it never got caught

Reply:
can we get a comment saying something to that effect?